cw-orch/src/prelude.rs

```diff
@@ -60,6 +60,7 @@ pub use crate::daemon::{
     DaemonAsyncBuilder,
     // expose the sync variants
     DaemonBuilder,
+    DeployedChains,
```

Comment:
not sure this needs to be in prelude. Is it available through regular mod?

Reply:
Yep, thanks!